### PR TITLE
Created validation docs

### PR DESCRIPTION
**Pull request by WilliamGawlik**
Created the following mongoDB collection validation docs
- Sets
- Exercises
- Workouts
- Users

### CODE REVIEW

**Review by JamesPRobertson — DISMISSED** [review]
Very nice! NOT

File: mongoDB/collections/sets_validation.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  $jsonSchema: {
+    required: [
+      'user_id',
+      'date',
+      'reps',
+      'weight',
+      'time',
+      'personal_record',
+      'exercise'
```
Comment (JamesPRobertson) [inline mongoDB/collections/sets_validation.json]: Exercise is not defined.

File: mongoDB/collections/sets_validation.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        maximum: 100000,
+        description: 'must be an integer in [ 0, 100000 ] and is required'
+      },
+      time: {
+        bsonType: 'int',
+        minimum: 0,
+        maximum: 100000,
+        description: 'must be a int in [ 0, 100000 ] and is required'
+      },
+      comment: {
```
Comment (JamesPRobertson) [inline mongoDB/collections/sets_validation.json]: Make comment a required property.

File: mongoDB/collections/measurements_validation.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          'imperial',
+          'metric'
+        ],
+        description: 'must be a enum in [ metric, imperial ] and is required'
+      },
+      manual_entry: {
+        bsonType: 'bool',
+        description: 'must be a bool and is required'
+      },
+      comment: {
```
Comment (JamesPRobertson) [inline mongoDB/collections/measurements_validation.json:43]: Make comment a required property.

**Review by JamesPRobertson — APPROVED** [review]
Looks good.

**Review by jverswijver — APPROVED** [review]

**Pull request merged**